tests/plugins/test_plugins_manager.py

```diff
@@ -73,95 +72,6 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-@pytest.mark.db_test
-class TestPluginsRBAC:
```

Comment:
Hmm, don't we need to keep these?  This is still in the plugin interface.

Reply:
This test requires a Flask application, that's why I removed it. In my opinion we should not have Flask related test in Airflow core. If you really want to test this, we can move it to FAB provider.

Reply:
Hmm, that makes me wonder if we need to refactor the pluginmanager to only know about fastapi apps, and keep all the flask/fab backcompat stuff isolated in the fab provider?

Reply:
Basically, having an untested interface on the pluginmanager doesn't feel great to me.

Reply:
> Well, it depends how strict you are with "the same way" and "working". Currently, the pages are accessible directly - no menu support - and no iframe hackiness yet either.

So ... those tests will not work anyway :) Let's nuke them and bring them back when we get to it maybe?

Reply:
Sure.